src/main/java/uk/gov/nationalarchives/pdi/step/jena/serializer/JenaSerializerStep.java

```diff
@@ -93,7 +93,13 @@ public boolean processRow(final StepMetaInterface smi, final StepDataInterface s
         if (row == null) {
             // serialize the jena model
             try {
+                final long start = System.currentTimeMillis();
+                logBasic("Starting serialization for: {0}", getStepname());
+
                 serializeModel(meta, data);
+
+                final long end = System.currentTimeMillis();
+                logBasic("Completed serialization for: {0} in: {1} ms", getStepname(), Long.toString(end - start));
```

Comment:
`StopWatch` is quite nice for this sort of thing: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/time/StopWatch.html